parts/k8s/kubernetesmasterresources.t

```diff
@@ -297,11 +291,9 @@
         ,
         "enableIPForwarding": true
 {{end}}
-{{if .MasterProfile.IsCustomVNET}}
```

Comment:
Before: if custom VNET attach a NSG to the master NIC

After: attach a NSG to the master NIC always